quickstart.md

```diff
@@ -0,0 +1,9 @@
+# Quickstart Guide
+
+Clone the repo, then open the directory in terminal. Run both these commands in parellel (in two terminals).
+```cargo run --bin fcos-graph-builder -- -c dist/fcos-graph-builder.toml.sample```
+and
+```cargo run --bin fcos-policy-engine -- -c dist/fcos-policy-engine.toml.sample```
```

Comment:
Clone the repo, then open the directory in terminal. Run both these commands in parallel (in two terminals):

```
cargo run --bin fcos-graph-builder -- -c dist/fcos-graph-builder.toml.sample

cargo run --bin fcos-policy-engine -- -c dist/fcos-policy-engine.toml.sample
```